src/install/aim/agent-lang/python-config.mdx

```diff
@@ -0,0 +1,71 @@
+---
+componentType: default
+headingText: Configure the Python agent
+---
+
+You installed the Python agent in the last step, but you still need to make configurations so it recognizes and collects AI-specific data. For AI data, you need to enable the `ai_monitoring` and `ai_monitoring.streaming` flags. Both prompt the Python agent to expose AI data within your app data. 
+
+1. We recommend applying these configurations from `newrelic.ini` config file. If you have a more complex set up with multiple environments, you can configure with environment variables.
+
+    <CollapserGroup>
+        <Collapser
+            id="config-file"
+            title="Enable with config file"
+        >
+
+    ```yaml
+    ai_monitoring.enabled = true
+    ai_monitoring.streaming = true
+    ```   
+        </Collapser>
+        <Collapser
+            id="config-enviro"
+            title="Enable with environment variables"
+        >
+
+    ```bash
+    NEW_RELIC_AI_MONITORING_ENABLED=true
+    NEW_RELIC_AI_MONITORING_STREAMING=true
+    ```
+
+        </Collapser>
+    </CollapserGroup>
+
+3. One instance of the Python agent sends a maximum of 100,000 machine learning (ML) events per minute. Once the agent hits that maximum, it samples your ML events, giving a general representation of events across the harvest cycle. 
+
+    You can adjust this value to ELEPHANT.
+
+        <CollapserGroup>
+            <Collapser
+                id="config-file"
+                title="Enable with config file"
+            >
+
+        ```yml
```

Comment:
I'm a little lost to what this configuration does. This is my understanding:

- The agent captures every single ML event until it hits 100,000 events
- The agent automatically samples after this
- What are customers setting here? Are they setting the number of samples the agent should collect? Is it a time interval? Is there a max and min value? If there is a max value, why would a customer choose a lower value over the max?

Reply:
I think the `event_harvest_config.harvest_limits.ml_event_data` below is based on an older approach to manage AI events. `event_harvest_config.harvest_limits.custom_event_data` is the current version. 

If you set this this value to 100k, we would collect up to 100k events per harvest cycle before dropping events. 100k is a very large number in this context so that essentially means no sampling will happen for ai events (or any event). Setting it to a lower value like 1k or less could mean sampling depending on the volume of the app.

Reply:
A customer who's being careful about cost or just doesn't need that much data in NR might choose to set this to a lower value